### PR TITLE
exlm-287-styles

### PR DESCRIPTION
**Pull request by adobenitesh**
Jira ID: EXLM-287

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/docs/integrations-learn/experience-cloud/solution-categories/content-management
- After: https://bugfix-exlm-287-styles--exlm--adobe-experience-league.hlx.page/docs/integrations-learn/experience-cloud/solution-categories/content-management

### CODE REVIEW

File: blocks/header/header.scss [hunk 1]
```diff
@@ -63,7 +64,7 @@
   font-weight: initial;
 }
 
-@mixin chevron($thickness: 2px) {
+@mixin chevron($thickness: 1.5px) {
```
Comment (ahmed-musallam) [inline blocks/header/header.scss:67]: Let's not use fractions, please use 2px here.

File: blocks/header/header.scss [hunk 2]
```diff
@@ -518,6 +525,15 @@ header {
 
         > ul {
           flex-direction: column;
+
+          & li > a {
```
Comment (ahmed-musallam) [inline blocks/header/header.scss:529]: There's already a hover state for these items by class, is this neccessary?

**Review by ahmed-musallam — CHANGES_REQUESTED** [review]

File: blocks/header/header.scss (hunk 2 above)
Comment (adobenitesh) [reply blocks/header/header.scss:529]: this is for matching the background-color and border-radius values of submenus items under Learn/Community tab(to fix one of the QA defects)

File: blocks/header/header.scss (hunk 1 above)
Comment (adobenitesh) [reply blocks/header/header.scss:67]: sure I will revert to 2px, however 2px value is making arrow bit thicker and as per QA that is not matching with XD, this is related to discussion we had today for XD match consideration I have also updated EXLM-287 with details.

File: blocks/header/header.scss (hunk 2 above)
Comment (ahmed-musallam) [reply blocks/header/header.scss:506]: Can you elaborate? what's special about these items?

**Review by ahmed-musallam — CHANGES_REQUESTED** [review]

**Review by ahmed-musallam — APPROVED** [review]

File: blocks/header/header.scss (hunk 2 above)
Comment (adobenitesh) [reply blocks/header/header.scss:506]: this fix is related to defect no 11 (web) (EXLM-287)
- Hovering style color for sublinks inside learn and community tab should match with XD

image

Comment (thomsebastin) [reply blocks/header/header.scss:506]: I am merging this PR to move the ticket. @adobenitesh Please discuss this item with @ahmed-musallam and let's handle in a separate PR if a change is required.

**Pull request merged**